app/src/test/java/com/arielwang/workoutlogger/track/TrackViewModelTest.kt

```diff
@@ -95,8 +92,9 @@ class TrackViewModelTest {
         }
     }
 
+    // check setDefaultTextFieldValue function in weightSection TextFiled
```

Comment:
I think the function name is good enough to explain these tests. Don't need additional comments here